ui/app/AppLayouts/Wallet/popups/AddAccountWithPrivateKeyModal.qml

```diff
@@ -81,17 +81,29 @@ ModalPopup {
         validationError: popup.passwordValidationError
     }
 
-
     StyledTextArea {
         id: accountPKeyInput
+        customHeight: 88
         anchors.top: passwordInput.bottom
         anchors.topMargin: marginBetweenInputs
-        //% "Paste the contents of your private key"
-        placeholderText: qsTrId("paste-the-contents-of-your-private-key")
+        anchors.left: parent.left
+        anchors.right: parent.right
+        validationError: popup.privateKeyValidationError
         //% "Private key"
         label: qsTrId("private-key")
-        customHeight: 88
-        validationError: popup.privateKeyValidationError
+        textField.wrapMode: Text.WordWrap
+        textField.horizontalAlignment: TextEdit.AlignHCenter
+        textField.verticalAlignment: TextEdit.AlignVCenter
+        textField.font.pixelSize: 15
+        textField.font.weight: Font.DemiBold
+        //% "Paste the contents of your private key"
+        placeholderText: qsTrId("paste-the-contents-of-your-private-key")
+        textField.placeholderTextColor: Style.current.secondaryText
+        textField.selectByMouse: true
+        textField.selectByKeyboard: true
+        textField.selectionColor: Style.current.secondaryBackground
+        textField.selectedTextColor: Style.current.secondaryText
+        textField.color: Style.current.textColor
```

Comment:
some of these are already default values in the StyledTextArea, can you only change the ones that need customisation

Reply:
Updated! good catch